tests/ops/integration_tests/test_connection_configuration_integration.py

```diff
@@ -1425,3 +1426,29 @@ def test_test_connection(
         connector = get_connector(rds_mysql_connection_config)
         with pytest.raises(ConnectionException):
             connector.test_connection()
+
+
+@pytest.mark.integration_external
+@pytest.mark.integration_rds_postgres
+class TestRDSPostgresConnector:
+    def test_connector(
+        self,
+        rds_postgres_connection_config,
+    ) -> None:
+        connector = get_connector(rds_postgres_connection_config)
+        assert connector.__class__ == RDSPostgresConnector
+        assert connector.rds_client
+
+    def test_test_connection(
+        self,
+        db: Session,
+        rds_postgres_connection_config,
+    ) -> None:
+        connector = get_connector(rds_postgres_connection_config)
+        assert connector.test_connection() == ConnectionTestStatus.succeeded
+
+        rds_postgres_connection_config.secrets["aws_secret_access_key"] = "bad_key"
+        rds_postgres_connection_config.save(db)
+        connector = get_connector(rds_postgres_connection_config)
+        with pytest.raises(ConnectionException):
+            connector.test_connection()
```

Comment:
a bit of a nitpick, but maybe we can move this to a separate test ? e.g `test_connection_with_bad_key` or something, I find it a bit easier to read / maintain when each test only tests for one thing